config/spotbugs-exclude.xml

```diff
@@ -73,6 +83,10 @@
       <Bug pattern="BC_UNCONFIRMED_CAST_OF_RETURN_VALUE"/>
     </Or>
   </Match>
+  <Match>
+    <!-- Till https://github.com/mebigfatguy/fb-contrib/issues/309 -->
+    <Bug pattern="CE_CLASS_ENVY"/>
```

Comment:
This pattern violates methods like [this](
https://github.com/checkstyle/checkstyle/blob/399f1b48b79731d65e824eb927971ef6174c3463/src/main/java/com/puppycrawl/tools/checkstyle/JavaParser.java#L225)